src/installer/tests/Assets/TestUtils/TestProjects.props

```diff
@@ -5,7 +5,7 @@
   -->
 
   <PropertyGroup>
-    <NetCoreAppCurrent>net8.0</NetCoreAppCurrent>
+    <NetCoreAppCurrent>net9.0</NetCoreAppCurrent>
```

Comment:
Why not to use NetCoreAppCurrentVersion?

Reply:
These projects don't import the repo infrastructure.